### PR TITLE
Set browser detection to be env-var-driven

### PR DESCRIPTION
**Pull request by gavinorland**
• If `DETECT_BROWSER` is not defined, or set as anything other than 'true', then `BROWSER_CONFIG` may or may not be defined - there is no detection.
• If `DETECT_BROWSER` is defined and set as 'true', then `BROWSER_CONFIG` must also be defined (in the expected format).

### CODE REVIEW

**Comment by pixie79** [comment]
um, why do we need a config flag for this?

**Comment by gavinorland** [comment]
@pixie79 Richard asked for this to be config-driven so we can easily and quickly set the rules and turn it on or off (since it could potentially have quite a drastic effect).

**Comment by shepda** [comment]
The second on/off flag is there so it's easy to turn the feature on/off without losing the browser version config which would happen if we set an empty browser version config to turn it off.

File: .drone.yml [hunk 1]
```diff
@@ -154,6 +154,10 @@ pipeline:
         target: ANALYTICS_URL
       - source: DEV_ANALYTICS_SITE_ID
         target: ANALYTICS_SITE_ID
+      - source: DEV_WWW_BROWSER_CONFIG
```
Comment (shepda) [inline .drone.yml]: Perhaps we should call this `DEV_WWW_BROWSER_VERSIONS` rather than `DEV_WWW_BROWSER_CONFIG` as it more closely describes what it is?

Comment (gavinorland) [reply .drone.yml]: I've renamed that as far as the client is concerned. I think this PR can sit here a little while now, pending Belinda making a few more changes.

**Comment by gavinorland** [comment]
@pixie79  Are you okay with this going through now?

**Pull request merged**